.github/workflows/multi-node.yml

```diff
@@ -0,0 +1,63 @@
+name: Multi node test
+on:
+  # pull_request:
+  push:
+    # branches:
+    #   - master
+    # tags-ignore: [v.*]
+jobs:
+  run-multi-node-tests:
+    name: Multi Node Test
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix: # *DO NOT EDIT* the following line is generated by running ./scripts/gh-generate-projects-matrix.sh from the root folder
+        project: [akka-actor, akka-actor-testkit-typed, akka-actor-typed, akka-cluster, akka-cluster-metrics, akka-cluster-sharding, akka-cluster-sharding-typed, akka-cluster-tools, akka-cluster-typed, akka-coordination, akka-discovery, akka-distributed-data, akka-serialization-jackson, akka-multi-node-testkit, akka-osgi, akka-persistence, akka-persistence-query, akka-persistence-typed, akka-persistence-testkit, akka-protobuf, akka-protobuf-v3, akka-pki, akka-remote, akka-slf4j, akka-stream, akka-stream-testkit, akka-stream-typed, akka-testkit, akka-actor-tests, akka-actor-typed-tests, akka-bench-jmh, akka-docs, akka-bill-of-materials, akka-persistence-shared, akka-persistence-tck, akka-persistence-typed-tests, akka-remote-tests, akka-stream-tests, akka-stream-tests-tck]
```

Comment:
Do all of those have multi-node tests?

Reply:
no, let me see if I can find a way to do an automatic exclusion/inclusion